results_schema/factories.py

```diff
@@ -0,0 +1,95 @@
+import factory 
+import factory.fuzzy
+from results_schema import schema
+import uuid
+from datetime import datetime
+import testing.postgresql
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker, scoped_session, create_session
+
+sessionmaker = sessionmaker()
+session = scoped_session(sessionmaker)
+
+class ExperimentFactory(factory.alchemy.SQLAlchemyModelFactory):
+    class Meta:
+        model = schema.Experiment
+        sqlalchemy_session = session
+    experiment_hash = factory.fuzzy.FuzzyText()
+    config = {}
+
+class ModelGroupFactory(factory.alchemy.SQLAlchemyModelFactory):
+    class Meta:
+        model = schema.ModelGroup
+        sqlalchemy_session = session
+    model_type = 'sklearn.ensemble.RandomForestClassifier'
```

Comment:
Yes, these are default parameters for testing. All these default variables are just examples of valid, reasonable values for these objects.

So if you want to create a ModelGroup object for testing, it will use the RandomForestClassifier unless you override it (ie, ModelGroupFactory(model_type='sklearn.ensemble.SomeOtherClassifier').

Point taken on the testing focus. I could add some language to the README that more specifically talks about testing, and add a docstring to the factories module that talks about this too.